nni/retiarii/hub/pytorch/autoformer.py

```diff
@@ -267,87 +277,141 @@ class AutoformerSpace(nn.Module):
         The scaler on score map in self-attention.
     rpe : bool
         Whether to use relative position encoding.
-
     """
 
     def __init__(
-            self,
-            search_embed_dim: Tuple[int, ...] = (192, 216, 240),
-            search_mlp_ratio: Tuple[float, ...] = (3.5, 4.0),
-            search_num_heads: Tuple[int, ...] = (3, 4),
-            search_depth: Tuple[int, ...] = (12, 13, 14),
-            img_size: int = 224,
-            patch_size: int = 16,
-            in_chans: int = 3,
-            num_classes: int = 1000,
-            qkv_bias: bool = False,
-            drop_rate: float = 0.,
-            attn_drop_rate: float = 0.,
-            drop_path_rate: float = 0.,
-            pre_norm: bool = True,
-            global_pool: bool = False,
-            abs_pos: bool = True,
-            qk_scale: Optional[float] = None,
-            rpe: bool = True,
+        self,
+        search_embed_dim: Tuple[int, ...] = (192, 216, 240),
```

Comment:
Did you feel `search_embed_dim` to be a little weird name? Maybe `embed_dim` is better and neater?

Reply:
This name is following the previous code.

Reply:
If you feel okay with it, you can keep it.